### DISCUSSION TITLE
More Google services for EDC

### DISCUSSION THREAD

**Discussion by git-masoud** [eclipse-edc/.github#24] [Ideas]
Hello Team,

I would like to start a discussion about potentially adding more Google services to EDC, such as BigQuery and Google Secret Manager. Given our collaboration with Google developers, we aim to contribute more to the EDC with new features.

Firstly, the Google Secret Manager as a EDC vault which provides a secure and centralized way to store and manage the sensitive data. This is missing in the EDC vaults.

Secondly, BigQuery as a data-plane which is a powerful cloud-based data warehousing and analytics solution that can help users to process and analyze large datasets more efficiently. 

I believe that Incorporating Google services into the EDC will make it more self-sufficient for users deploying the EDC on GCP, reducing the reliance on other cloud services. Furthermore, by adding these services to the upstream repository, we can help other developers who are working on similar projects to benefit from these tools.

However, before making any decisions, I would like to hear from the rest of the team. Do you think that adding these services to EDC would be ok? Are there any concerns or challenges that we need to consider before making a decision?

Best regards,
Masoud

**Comment by jimmarino** [comment]
These sound like they would be great additions to EDC. We are currently discussing moving out cloud-specific extensions into separate repositories, as detailed in this decision record:

https://github.com/eclipse-edc/Connector/pull/2597

Assuming we reach a consensus on the DR (the technical committers have already discussed it), a Google repository under the "Eclipse EDC" organization would be created (similar to the Identity Hub and Federated Catalog). 

The question the committers will have is (which we don't need to answer here), "are your organization and Google signing up to maintain that repository and releases from it long-term?" This will be a pre-requisite to avoid code dumps and unmaintained code.
